Rules/Strings.resx

```diff
@@ -897,6 +897,18 @@
   <data name="AvoidTrailingWhitespaceError" xml:space="preserve">
     <value>Line has trailing whitespace</value>
   </data>
+  <data name="AvoidLongLinesName" xml:space="preserve">
+    <value>AvoidLongLines</value>
+  </data>
+  <data name="AvoidLongLinesCommonName" xml:space="preserve">
+    <value>Avoid long lines</value>
+  </data>
+  <data name="AvoidLongLinesDescription" xml:space="preserve">
+    <value>Each line should be not be too long.</value>
```

Comment:
```suggestion
    <value>Line lengths should be less than the configured maximum</value>
```

Reply:
This file seems to be inconsistent with `.` -- might defer to @bergmeister on that one

Reply:
Consistency would be nice if possible, but I don't have a strong opinion on this

Reply:
Changed. I'm not a fan of `.`s in one sentence messages, but I'll defer to whatever you guys prefer.